locale/uk/foundation/members.md

```diff
@@ -96,6 +96,9 @@ Node.js Foundation відкритий для бізнесу та окремих
 <a target="_new" class="imagelink" href="//yld.io">
   <img alt="yld!" class="memberlogo" src="/static/images/foundation/yldLogo.png" />
```

Comment:
The ! weren't consistent. The only other alt that held it was Yahoo so I assumed it was literally a typo.